src/TypeSpec.Extension/Emitter.Csharp/package.json

```diff
@@ -40,6 +40,7 @@
     "peerDependencies": {
         "@azure-tools/typespec-azure-core": ">=0.36.0 <1.0.0",
         "@azure-tools/typespec-client-generator-core": ">=0.36.0 <1.0.0",
+        "@azure-tools/typespec-azure-rulesets": ">=0.36.0 <1.0.0",
```

Comment:
Please make the same change in microsoft/typespec so we stay in sync

Reply:
PR Created: https://github.com/microsoft/typespec/pull/3518